Library/Homebrew/shims/super/mig

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
+
 pwd="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
It seems that there are multiple styles for resolving the current directory. I'm wondering should we make them consistence as well?